docs/guides/manage-cost.mdx

```diff
@@ -70,7 +93,7 @@ Additionally, you can determine cost per instance or per job at any time.
 
 ### View instance cost
 
-To determine how much has been billed to an instance during the current billing cycle, from the [Instances page](https://cloud.ibm.com/quantum/instances), click the instance to open its details page.
+To determine how much has been billed to an instance during the current billing cycle, from the <LegacyContent>[IBM Cloud Instances page](https://cloud.ibm.com/quantum/instances)</LegacyContent><CloudContent>[Instances page](https://quantum.cloud.ibm.com/instances)</CloudContent>, click the instance to open its details page.
```

Comment:
I think we have to re-phrase this section because we are not showing this information (yet). As an alternative we can say they can see their usage (not cost) in the [analytics section](https://quantum.cloud.ibm.com/admin/analytics).